sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/sampleoverview/SampleOverviewView.groovy

```diff
@@ -29,6 +29,14 @@ class SampleOverviewView extends VerticalLayout implements HasHotbar, HasTitle {
         this.setMargin(false)
     }
 
+    /**
+     * Returns the {@link ProjectSamplesController} of this view
+     * @return ProjectSamplesController
+     */
+    ProjectSamplesController getController(){
+        return projectSamplesController
+    }
+
```

Comment:
As the SampleOverviewView does not use this controller, it can be removed from this class and injected into the other class directly. This reduces complexity and removes unnecessary dependencies and public methods.

Reply:
What function does the SampleOverviewView have then? Do we really need it? I think there is one layer too much. 
I think the best way is to keep it as is or remove the SampleOverviewView.

Reply:
It provides the hotbar and the title. Here SRP is applied. One component is responsible for display the other for the layouting

Reply:
I removed this class now and with that also the controller from the view :)